<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelDownload</class>
 <widget class="QWidget" name="ModelDownload">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Model Download</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#ModelDownload {
    background: #1a1412;
}

QLabel {
    background: transparent;
    font-family: &quot;Press Start 2P&quot;, &quot;Courier New&quot;, monospace;
}

QGroupBox {
    background: transparent;
    border: 2px solid #2a1d19;
    border-radius: 0px;
    margin-top: 16px;
    padding-top: 12px;
    padding-left: 12px;
    font-size: 10px;
    letter-spacing: 0px;
    text-transform: uppercase;
    color: #8d7b68;
    font-weight: bold;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 8px;
    top: -8px;
    padding: 0 4px 0 4px;
    background: #1a1412;
}

QScrollArea {
    background: transparent;
    border: none;
}

QScrollBar:vertical {
    background: #14100f;
    width: 12px;
    border: 2px solid #2a1d19;
    border-radius: 0px;
}

QScrollBar::handle:vertical {
    background: #3d2b25;
    border: 1px solid #5a4339;
    border-radius: 0px;
    min-height: 30px;
}

QScrollBar::handle:vertical:hover {
    background: #5a4339;
}

QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
    height: 0px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="contentWidget">
     <property name="styleSheet">
      <string notr="true">QWidget#contentWidget {
    background: #1a1412;
}</string>
     </property>
     <layout class="QVBoxLayout" name="contentLayout">
      <property name="spacing">
       <number>24</number>
      </property>
      <property name="leftMargin">
       <number>48</number>
      </property>
      <property name="topMargin">
       <number>40</number>
      </property>
      <property name="rightMargin">
       <number>48</number>
      </property>
      <property name="bottomMargin">
       <number>40</number>
      </property>
      <item>
       <widget class="QLabel" name="titleLabel">
        <property name="text">
         <string>MODEL DOWNLOAD</string>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #c4a77d;
    font-size: 18px;
    font-weight: bold;
    letter-spacing: 1px;
    background: transparent;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="modelsGroup">
        <property name="title">
         <string>AVAILABLE MODELS</string>
        </property>
        <layout class="QVBoxLayout" name="modelsLayout">
         <property name="spacing">
          <number>12</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>12</number>
         </property>
         <item>
          <widget class="QScrollArea" name="modelsScrollArea">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>1</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumHeight">
            <number>200</number>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">QScrollArea {
    border: none;
}</string>
           </property>
           <widget class="QWidget" name="modelsContainer">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>800</width>
              <height>300</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
           </property>
            <layout class="QVBoxLayout" name="modelsContainerLayout">
             <property name="spacing">
              <number>12</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>80</number>
             </property>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="buttonLayout">
        <property name="spacing">
          <number>12</number>
         </property>
        <item>
         <widget class="QPushButton" name="btnDownloadAll">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>36</height>
           </size>
          </property>
          <property name="text">
           <string>DOWNLOAD ALL</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background: #3d2b25;
    color: #e8d5c4;
    border: 2px solid #5a4339;
    border-radius: 0px;
    font-size: 11px;
    font-weight: bold;
    letter-spacing: 0px;
    padding: 6px 12px;
    text-align: center;
}

QPushButton:hover {
    background: #5a4339;
    color: #c4a77d;
    border-color: #c4a77d;
}

QPushButton:pressed {
    background: #c4a77d;
    color: #1a1412;
    border-color: #c4a77d;
}

QPushButton:disabled {
    background: #1a1412;
    color: #6b5c52;
    border-color: #2a1d19;
}</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnRefresh">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>36</height>
           </size>
          </property>
          <property name="text">
           <string>REFRESH</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background: #1a1412;
    color: #c4a77d;
    border: 2px solid #3d2b25;
    border-radius: 0px;
    font-size: 11px;
    font-weight: bold;
    letter-spacing: 0px;
    padding: 6px 12px;
    text-align: center;
}

QPushButton:hover {
    background: #3d2b25;
    color: #e8d5c4;
    border-color: #c4a77d;
}

QPushButton:pressed {
    background: #c4a77d;
    color: #1a1412;
    border-color: #c4a77d;
}

QPushButton:disabled {
    color: #6b5c52;
    border-color: #2a1d19;
}</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
